Makefile

```diff
@@ -149,8 +149,8 @@ test-sim-multi-seed-short: runsim
 	@$(BINDIR)/runsim -Jobs=4 -SimAppPkg=$(SIMAPP) -ExitOnFail 50 5 TestFullAppSimulation
 
 test-sim-deterministic: runsim
-	@echo "Running short multi-seed application simulation. This may take awhile!"
-	@$(BINDIR)/runsim -Jobs=4 -SimAppPkg=$(SIMAPP) -ExitOnFail 1 1 TestAppStateDeterminism
+	@echo "Running application deterministic simulation. This may take awhile!"
+	@$(BINDIR)/runsim -Jobs=1 --Seeds="4003615766033486007" -SimAppPkg=$(SIMAPP) -ExitOnFail 1 1 TestAppStateDeterminism
```

Comment:
`4003615766033486007` was discovered in local tests to create a reproducible failure

Reply:
I will restore this part